components/cam/src/physics/crm/sam/ADV_MPDATA/advect_scalar.F90

```diff
@@ -1,7 +1,9 @@
 module advect_scalar_mod
   use advect_scalar2D_mod
   use advect_scalar3D_mod
+#if defined(_OPENACC)
```

Comment:
Why do this here? I though `#if defined(_OPENACC)` was done inside openacc_utils. This requires `#ifdef` everywhere we use it whereas having it inside `openacc_utils` looks a little cleaner.